inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/reduce_ops.cpp

```diff
@@ -75,9 +73,9 @@ const std::vector<ngraph::helpers::ReductionType> reductionLogicalTypes = {
 const auto paramsOneAxis = testing::Combine(
         testing::Values(std::vector<int>{0}),
         testing::ValuesIn(opTypes),
-        testing::Values(true, false),
+        testing::ValuesIn(keepDims),
         testing::ValuesIn(reductionTypes),
-        testing::Values(InferenceEngine::Precision::FP32),
+        testing::Values(netPrecisions[0]),
```

Comment:
`ValuesIn(netPrecisions)`

Reply:
This is to specifically test one axis, for checking that the op works with different precisions is in [here](https://github.com/openvinotoolkit/openvino/pull/5557/files#diff-031b9844c88a4dee1de585559d931fe92a0a5b28f47321f4ef42f61bfeec4481L101). The idea is not to make for every single test several precisions as it is checked in different test case.